fixtures/no-killable

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+'use strict';
+
+process.on('SIGTERM', () => {
+  console.log('got sigterm but we ignore it');
+});
+
+process.send('');
+
+setInterval(() => {
+  // Run forever
+}, 20000);
```

Comment:
This is not forever. Should be much higher in case of very slow CI's.

Reply:
The number value isn't important since we use an `setInterval` here and the program never finishes.